changelogs/fragments/8444-fix-redfish-gen2-upgrade.yaml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - wdc_redfish_command - minor change to handle upgrade file for redfish gen 2 platform (https://github.com/ansible-collections/community.general/pull/8444).
```

I think Redfish is usually capitalized? (No idea about `gen 2`.)
```suggestion
  - wdc_redfish_command - minor change to handle upgrade file for Redfish gen 2 platform (https://github.com/ansible-collections/community.general/pull/8444).
```

---

Gen 2 upgraded version of WD Redfish based enclosure

---

> I think Redfish is usually capitalized? (No idea about `gen 2`.)

Changes done and pushed.